.env.development

```
@@ -0,0 +1 @@
+VITE_ENGINE_PORT=8080
```

Avoid coupling with the desktop app implementation.
Also this env is specific to the server, so if we keep that file at the root I'd at least prefix the variable name with the app it's intended for.
```suggestion
SERVER_PORT=8080
```